pkg/lib/flavor/image_flavor.go

```diff
@@ -28,9 +28,9 @@ func GetImageFlavor(label string, encryptionRequired bool, keyURL string, digest
 
 	var encryption *model.Encryption
 
-	description := model.Description{
-		Label:      label,
-		FlavorPart: "IMAGE",
+	description := map[string]interface{}{
```

Comment:
would this cause any backward compatibility issue on WLS?

Reply:
Analysed the wls code and the GetImageFlavor method used by wls is in sst\isecl\lib\flavor and since this method is used only by test files in intel-secl package, we have run all the unit test cases under lib package and found that this change does not cause any issues. Is that fine?